app/views/groups/index.html.erb

```diff
@@ -0,0 +1,238 @@
+        padding: 15px;
+        margin: 10px 0;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        flex: 1 1 auto;
+        overflow-y: auto;
+    }
+    
+
+    .category-block i {
+        font-size: 24px;
+        margin-bottom: 10px;
+    }
+
+    .category-block .category-details i {
+    font-size: 30px;
+    margin-top: 17px;
+    margin-left: 12px;
+}
+
+
+    .category-details {
+      display: flex;
+      justify-content: space-between;
+      align-items: center; 
+      width: 100%;
+      flex-direction: row; 
+  }
+  
+
+    .category-link {
+        text-align: center;
+        width: 100%
+        margin-top: 15px;
+    }
+
+    .horizontal-center {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 20px;  
+}
+
+.main-logo {
+    position: absolute;
+    bottom: -150px;
+    right: 160px;
+    width: 70px;
+    height: auto;
+}
+
+.user-name {
+    font-size: 15px; 
+    margin-left: 20px; 
+}
+
+.btn-custom {
+    background-color: #5fb523; 
+    color: white; 
+    border: 2px solid white; 
+    padding: 10px 18px; 
+    position: relative; 
+}
+
+.btn-custom::before {
+    content: ''; 
+    position: absolute; 
+    top: 2px;
+    left: 2px;
+    right: 2px;
+    bottom: 2px;
+    border: 1px solid white; 
+}
+
+.category-name {
+  font-family: 'Proxima Nova', sans-serif;
+  font-weight: 500;
+  font-size: 20px;
+}
+
+.category-total {
+  font-family: 'Proxima Nova', sans-serif;
+  font-size: 20px;
+  font-weight: 300;
+}
+
+.footer-logo {
+    flex-shrink: 0;
+    align-self: flex-end;
+    margin-top: auto;
+    margin-bottom: 20px;
+    padding: 20px 0;
+}
+
+</style>
+
+<div class="container">
+    <nav class="navbar navbar-expand-lg" style="background-color: #3778c2; padding: 20px 0">
+        <div class="login-nav">
+        <span class="navbar-brand user-name">
+        <%= current_user.name %>
+    </span>
+    
+            <span class="navbar-text">
+                Categories
+            </span>
+            <% if user_signed_in? %>
+            <%= button_to "Log Out", destroy_user_session_path, method: :delete, class: "login-btn" %>
+        <% end %>
+        </div>
+    </nav>
+
+    <div class="mt-3">
+        <% @groups.each do |group| %>
+          <div class="category-block">
+          <div class="category-details">
+              <i class="<%= group.icon %>" aria-label="<%= group.name %> Icon"></i>
+              <h3 class="category-name"><%= group.name %></h3>
+              <p class="category-total">$ <%= group.entities.sum(:amount) %></p>
+          </div>
+          <div class="category-link">
+              <%= link_to 'View Transactions', group_entities_path(group) %>
+          </div>
+      </div>
+      
+        <% end %>
+
+        <!-- Utilizando la clase para centrar "Add a new category" -->
+        <div class="horizontal-center">
+        <%= link_to 'ADD CATEGORY', new_group_path, class: "mt-3 btn btn-custom" %>
```

Comment:
hi @TedLivist, thanks for the review.

I added it that way because it's that size that fixes the way I think my application needs, I see it that way because there is no requirement about the size of the buttons, just the colors, typography (font face, size and weight) and layout, it says that it's just a guideline, not to replicate the design.

#### Design
  - You should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
  -- Colors.
  -- Typography: font face, size and weight.
  -- Layout: composition and space between elements.

Reply:
Hi @rudicarrilloypr, I believe "**Layout: ...space between elements** and **Typography: ...size**" covers the changes I requested. You are not required to replicate the design exactly, but the layout and placement of elements should still be maintained.

If you are in doubt or disagree with the changes you can request additional review from a senior code reviewer using [this form](https://airtable.com/shrQAqnBwek5a0O0s).